examples/server_profiles/main.tf

```diff
@@ -275,10 +275,5 @@ resource "oneview_server_profile" "SPWithDL" {
       value = "Utc"
     }
   }
-  boot_mode {
```

Comment:
Can i know the reason for deleting this block?

Reply:
Its a duplicate attribute in the example file.